specs/common/knative-serving-conformance-testing-workflow.md

```diff
@@ -0,0 +1,36 @@
+# Knative Conformance Workflow
+
+This document covers the workflow steps to gain knative conformance
+
+## Conformance Steps
+1. Run Manual Tests
+2. Open Github PR in knative/specs repo 
+  - Use PR template below
+  - place in the following directory structure
+    - conformance/tests/results/ directory
+    - conformance/test/results/$vendor directory with a metadata file and test results
+    - ex; .../v1.0/2021-12-18-ibm-code-engine/...
```

Comment:
This has a version string in it, but the earlier text doesn't. Do we want the version in there? I think Doug was a "yes".

Reply:
That's the end-result of this comment: https://docs.google.com/document/d/1_gWhCeaU42DtHFkwVJOAbLVyuVaW5EhRkulLqQSTxkc/edit?disco=AAAARIwP9f4

Reply:
The end result from the comments and suggested edits was yes, though I didn't show multiple examples.